src/Form/Type/AdminType.php

```diff
@@ -173,7 +147,7 @@ protected function getFieldDescription(array $options)
     /**
      * @return AdminInterface
      */
-    protected function getAdmin(array $options)
+    private function getAdmin(array $options)
```

Comment:
```suggestion
    private function getAdmin(array $options): AdminInterface
```
Then, the `@return` docblock can be removed.

Reply:
Fixed